src/omnisharp/server.ts

```diff
@@ -393,149 +389,66 @@ export abstract class OmnisharpServer {
 
     // --- requests et al
 
-    public makeRequest<TResponse>(path: string, data?: any, token?: vscode.CancellationToken): Promise<TResponse> {
+    public makeRequest<TResponse>(command: string, data?: any, token?: vscode.CancellationToken): Promise<TResponse> {
 
         if (this._getState() !== ServerState.Started) {
             return Promise.reject<TResponse>('server has been stopped or not started');
         }
 
+        console.log(`makeRequest: command=${command}`);
+
         let startTime: number;
         let request: Request;
 
         let promise = new Promise<TResponse>((resolve, reject) => {
             startTime = Date.now();
 
             request = {
-                path,
+                command,
                 data,
                 onSuccess: value => resolve(value),
```

Comment:
Yeah, it does confuse tslint if I just pass the functions without creating lambdas -- "A Promise was found that appears to not have resolve or reject invoked on all code paths".

I suppose it's already been this way for several months already. I'll leave it as is for now.